src/app/auth/components/login.component.ts

```diff
@@ -49,6 +55,19 @@ export class LoginComponent implements OnInit {
     this.showTokenExp = environment.showTokenExpirationCustomization;
   }
 
+  ngAfterViewInit() {
+    if (!this.loggedIn) {
```

Comment:
The reason header component has all this logic about whether the user is logged in or not is because both situations are possible. However, this is not the case with login component. It will only show up when the user is not logged in.
You can safely remove lines 15, 59-64, 66-70.